pyproject.toml

```diff
@@ -55,7 +55,7 @@ pytest-mock = ">=3.11.1"
 pytest-xdist = ">=3.3.1"
 ruff = "0.0.282" # the ruff project is pretty early, even a patch bump tends to break stuff
 types-requests = "^2.31.0.2"
-vunnel = "^0.15.2"
+vunnel = "^0.22.2"
```

Comment:
Why does the `grype-db` pyproject need vunnel? I think grype-db just does `docker run vunnel:latest` now. Is this so that it's available in development shells?

Reply:
Nice callout! Removed vunnel.